doc/pages/administrator/062-handling_certificates.md

```diff
@@ -0,0 +1,114 @@
+# Handling Astarte certificates
+
+Astarte heavily requires SSL in a number of interactions, even though this can be bypassed with
+`ssl: false`. In general there are two main scenarios when dealing with certificates: either you
+already purchased SSL certificates for your domains or you might want the task to be handled by
+Let's Encrypt through cert-manager.
+
+The two alternative procedures for securing your Astarte deployment are outlined in the following
+sections.
+
+## Use your own certificates
+
+If you already own certificates for your domains, all it's needed is creating a TLS secret in the
+namespace in which Astarte resides. Assuming that the certificate and key are saved respectively as
+`cert.pem` and `privkey.pem`, simply run:
```

Comment:
when using voyager, `cert.pem` had to contain both the certificate and the CA bundle in a specific order. I think that's true even with the NGINX ingress. Should we add this to the doc?